persistence/src/main/java/edu/unc/lib/dl/persist/services/deposit/DepositModelManager.java

```diff
@@ -0,0 +1,479 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.dl.persist.services.deposit;
+
+import static org.apache.jena.rdf.model.ModelFactory.createDefaultModel;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.Writer;
+import java.nio.file.DirectoryNotEmptyException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.concurrent.locks.Lock;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.commons.io.FileUtils;
+import org.apache.jena.query.Dataset;
+import org.apache.jena.query.Query;
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QueryExecutionFactory;
+import org.apache.jena.query.QueryFactory;
+import org.apache.jena.query.QuerySolution;
+import org.apache.jena.query.ReadWrite;
+import org.apache.jena.query.ResultSet;
+import org.apache.jena.rdf.model.Bag;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.tdb.TDBFactory;
+import org.apache.jena.tdb.transaction.TDBTransactionException;
+import org.apache.jena.update.UpdateAction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.util.concurrent.Striped;
+
+import edu.unc.lib.dl.exceptions.InterruptedRuntimeException;
+import edu.unc.lib.dl.exceptions.RepositoryException;
+import edu.unc.lib.dl.fedora.PID;
+
+/**
+ * Manager which provides synchronized access to a common deposit model, allowing
+ * multiple transformation jobs to write to it at the same time.
+ *
+ * @author bbpennel
+ */
+public class DepositModelManager {
+
+    private static final Logger log = LoggerFactory.getLogger(DepositModelManager.class);
+    private static final int DEPOSIT_LOCK_STRIPES = 5;
+
+    private Path tdbBasePath;
+    // Locks to prevent simultaneous attempts to get the same dataset
+    private Striped<Lock> depositLocker;
+
+    /**
+     * Construct a deposit model manager
+     * @param tdbBaseDir path to the tdb directory
+     */
+    public DepositModelManager(String tdbBaseDir) {
+        this(Paths.get(tdbBaseDir));
+    }
+
+    /**
+     * Construct and initialize a deposit model manager
+     * @param depositsPtdbBasePathath
+     */
+    public DepositModelManager(Path tdbBasePath) {
```

Comment:
JavaDoc and method signature don't match

Reply:
Javadoc is butchered. Let me fix that